### ISSUE TITLE
Off centered display

### ISSUE THREAD

**Issue by Windowissue** [bluemagic123/tModLoader#3347]
### Version

Latest 1.4-stable

### OS

MacOs

### Platform

Steam

### This bug affects

The display of Tmodloader

### Description

Whenever I open Tmodloader, it first opens with the little intro scene condensed into the upper left corner. When that screen is transitioning to the title screen, it goes all black and returns almost to how it should be, but still off centered.

### Log File

[client.log](https://github.com/tModLoader/tModLoader/files/11069828/client.log)

<img width="1440" alt="Screenshot 2023-03-25 at 1 56 14 PM" src="https://user-images.githubusercontent.com/128929094/227736207-8aa4be22-3a55-43aa-a18b-ff0f06eb032a.png">
<img width="1440" alt="Screenshot 2023-03-25 at 1 56 30 PM" src="https://user-images.githubusercontent.com/128929094/227736220-39ab8094-a28a-46f3-9e25-b26d5af4c49a.png">
<img width="1440" alt="Screenshot 2023-03-25 at 1 56 35 PM" src="https://user-images.githubusercontent.com/128929094/227736237-0cd81e96-3122-4be4-bd4b-0d3a2d9afa01.png">
<img width="1440" alt="Screenshot 2023-03-25 at 1 56 47 PM" src="https://user-images.githubusercontent.com/128929094/227736241-ff7764c1-236d-45df-b268-375e1e717879.png">
*note I disabled mods to try and fix the issue, but that doesn't have an effect

### Steps to reproduce

1. Get on Mac
2. Open Steam
3. Open Tmodloader

### Expected Behavior

Display appear normal

### Actual Behavior

Screen is almost normal, but it is still off centered and is fitted into upper left corner still.

### Reproduction frequency

Almost every time, except for yesterday. I opened it yesterday randomly and it displayed normally, it was still off at the start, but when the title screen came it was normal.

### Additional Information

Tmodloader initially appears with a different icon when I open it, and it’s also called “dotnet” whenever I hover over the icon.
Health bars are off centered

**Comment by JavidPack** [comment]
Is this still happening on 1.4.4 (the latest stable, 2023.06)

**Comment by CausticWarrior** [comment]
Can confirm it is still happening

**Comment by CausticWarrior** [comment]
the "dotnet" title when hovered over and also the offcentered opening screen. My screen is centered correctly after the initial screen tho.

**Comment by ovilerg** [comment]
yeah i had the same issue but i figured it out, its a little tedious but first you go into a world, you go to settings, then cycle through resolution and when you do that it goes normal on any resolution so i just set it back to max. hope this helps.
